src/NuGet/Microsoft.CodeAnalysis.Common.nuspec

```diff
@@ -34,6 +34,7 @@ Supported Platforms:
             <dependency id="System.Dynamic.Runtime" version="4.0.11-rc2-23910" />
```

Comment:
@eerhardt i thought the design enabled us to upgrade CoreClr without upgrading all of CoreFX at the same time.